### DISCUSSION TITLE
Can you already give an estimate on when updaten and deleting users trough "user management" wil be implemented

### DISCUSSION THREAD

**Discussion by JohnBerendsenQA** [Soluto/oidc-server-mock#138] [General]
Hi, In the readme I've red that updating and deleting users will be implemented in the future. Could you give an expected date/timewhen this will be ready for use? Does not need to be exact but I would like to know (if this is known on your site) which quarter and which year you expect it to be implemented.

Kind regards,

John Berendsen

**Comment by AleF83** [comment]
Hi @JohnBerendsenQA 
Addition of users is implemented. You can see the endpoint [here](https://github.com/Soluto/oidc-server-mock/blob/5048998f05010b645dd6fa6d8f9851023f3ba47d/src/Controllers/UserController.cs#L31) and an example of request [here](https://github.com/Soluto/oidc-server-mock/blob/5048998f05010b645dd6fa6d8f9851023f3ba47d/e2e/tests/custom-endpoints/user-management.spec.ts#L68).
